modules/substrate/src/lib.rs

```diff
@@ -172,15 +172,21 @@ decl_module! {
 		) -> DispatchResult {
 			ensure_operational::<T>()?;
 			let _ = ensure_signed(origin)?;
-			frame_support::debug::trace!("Got header {:?}", header);
+			let hash = header.hash();
+			frame_support::debug::trace!("Going to import header {:?}: {:?}", hash, header);
```

Comment:
I don't remember what the logs look like, what info do the `"{:?}," header` logs include?

Reply:
For our chains, it is regular auto-derived `Debug` implementation (i.e. it shows all struct fields).